modules/ROOT/nav.adoc

```diff
@@ -7,29 +7,26 @@
  ** xref:prereq-hardware.adoc[Hardware Prerequisites]
  ** xref:prereq-software.adoc[Software Prerequisites]
  ** xref:prereq-network.adoc[Network Prerequisites]
- ** xref:verify-nfs.adoc[NFS Prerequisites]
-  *** xref:troubleshoot-nfs.adoc[Troubleshoot NFS Errors]
  ** xref:prereq-gravity-check.adoc[Preinstallation Verification]
 * xref:install-workflow.adoc[Install Anypoint Platform PCE]
- ** xref:install-installer.adoc[Run the GUI Installer]
- ** xref:install-auto-install.adoc[Run the CLI Installer]
-* xref:config-workflow.adoc[Post-installation Tasks]
- ** xref:config-workflow.adoc[Close port 61009]
+* xref:config-workflow.adoc[Configure Anypoint Platform PCE]
+ ** xref:install-disable-local-user.adoc[Delete the Local User after Installing Anypoint Platform PCE]
+ ** xref:install-config-ldap-pce.adoc[Configure an External Identity Provider in Anypoint Platform PCE]
+ ** xref:access-management-SMTP.adoc[Configure SMTP in Anypoint Platform PCE]
  ** xref:access-management-dns.adoc[Configure DNS on Anypoint Platform PCE]
+ ** xref:verify-nfs.adoc[Configure NFS on Anypoint Platform PCE]
```

Comment:
Apologies for the confusion, `verify-nfs` needs to continue to be under `Product Prerequisites` as it was before.
We are missing a new page for configuring NFS.
Please move this back to `Product Prerequisites` and we can work on the NFS in a future PR.

Reply:
Got it - thank you!